make build runs into Error 64

When i try to use ```make build``` it runs into ```make: *** [build] Error 64```

See the terminal output:
```
Dropping unreachable code [./~/ng-annotate-loader/loader.js!./~/humane-js/humane.js:13,0]
Dropping unused variable u [./~/ng-annotate-loader/loader.js!./~/jsonlint/web/jsonlint.js:1,3198]
Dropping unused variable e [./~/ng-annotate-loader/loader.js!./~/jsonlint/web/jsonlint.js:1,6184]
Dropping unused variable f [./~/ng-annotate-loader/loader.js!./~/jsonlint/web/jsonlint.js:1,7587]
Side effects in initialization of unused variable jsonlint [./~/ng-annotate-loader/loader.js!./~/jsonlint/web/jsonlint.js:1,0]
Condition always true [./~/ng-annotate-loader/loader.js!./~/nprogress/nprogress.js:6,0]
Dropping unreachable code [./~/ng-annotate-loader/loader.js!./~/nprogress/nprogress.js:8,3]
Dropping unused variable _output [./~/ng-annotate-loader/loader.js!./~/papaparse/papaparse.js:222,0]
Dropping unused variable _fields [./~/ng-annotate-loader/loader.js!./~/papaparse/papaparse.js:223,0]
Child extract-text-webpack-plugin:
        + 2 hidden modules
Child extract-text-webpack-plugin:
        + 2 hidden modules
Child extract-text-webpack-plugin:
        + 2 hidden modules
Child extract-text-webpack-plugin:
        + 2 hidden modules
Child extract-text-webpack-plugin:
        + 2 hidden modules
Child extract-text-webpack-plugin:
        + 2 hidden modules
Child extract-text-webpack-plugin:
        + 2 hidden modules
Child extract-text-webpack-plugin:
        + 8 hidden modules
Child extract-text-webpack-plugin:
        + 7 hidden modules
usage: cp [-R [-H | -L | -P]] [-fi | -n] [-apvX] source_file target_file
       cp [-R [-H | -L | -P]] [-fi | -n] [-apvX] source_file ... target_directory
make: *** [build] Error 64
```

Any idea? Thanks

--- a/Makefile
+++ b/Makefile
@@ -10,7 +10,7 @@ run:
 
 build:
 	@NODE_ENV=production ./node_modules/webpack/bin/webpack.js -p --optimize-minimize --optimize-occurence-order --optimize-dedupe --progress
-	@cp -Rf build/* examples/blog/build/
+	@cp -Rf build examples/blog/
 	@echo "Files build/ng-admin.min.css and build/ng-admin.min.js updated (with minification)"
 
 test:

